content/blog/2023/2023-08-23-fairness-dataset-bias.md

```diff
@@ -0,0 +1,74 @@
++++
+author = "Žan Mervič"
+date = "2023-08-23"
+draft = false
+title = "Orange Fairness - Dataset Bias"
+type = "blog"
+thumbImage = "/blog_img/2023/2023-08-23-fairness-dataset-bias-thumb.png"
+frontPageImage = "/blog_img/2023/2023-08-23-fairness-dataset-bias-thumb.png"
+blog = ["fairness", "dataset bias"]
+shortExcerpt = "Orange now supports methods for detecting and mitigating bias in machine learning."
+longExcerpt = "In an era where AI drives decisions impacting real lives, fairness in machine learning is paramount. Take the `Adult` dataset, which shows discrepancies in salary predictions based on gender. Addressing such concerns, Orange introduces a fairness add-on. Using new widgets, users can identify and mitigate biases in their datasets or model predictions."
++++
+
+## Fairness in Machine Learning
+
+Artificial intelligence and machine learning are increasingly used in everyday decisions that deeply impact individuals. This includes areas like employment, court sentencing, and credit application approvals. It's crucial that these decision-making tools don't favor certain demographic groups over others.
+
+Bias in machine learning can stem from multiple sources. It might arise from skewed training data or the model's design. This bias can result in unequal outcomes: unprivileged groups might receive lower salaries despite having similar qualifications as privileged ones or face longer prison sentences for identical crimes. Such biases can significantly affect individuals' lives.
+
+For example, in the "Adult" dataset, a notable bias exists between male and female genders. When predicting if a person's salary is ">50K" or "<=50K", females tend to be classified as "<=50K" more frequently than their male counterparts with similar attributes. This disparity showcases the dataset's bias, which can lead to biased and unfair predictions.
```

Comment:
I know it might be obvious to some, but I miss a part that would explain why is a adult more biased than we'd like our model to be. What is wrong with modelling data as is. If I understand the reasoning behind it, our world is biased (thus our data is biased) and we'd like to make it less so (thus we use proposed techniques). I think you should discuss these aspects. Otherwise I am not convinced why would anyone need these techniques.